### ISSUE TITLE
examples

### ISSUE THREAD

**Issue by fitoprincipe** [ThomasTheSpaceFox/Desutezeoid#1]
Hi there! I am a lover of 'point-and-click' and also do some python.. is there any example to try out the engine?

**Comment by ThomasTheSpaceFox** [comment]
if you download the latest stable release, running DZU-ENG1.py will start up an included demo program, that demonstrates various engine features, as well as some plugins, such as screen filters, an animation system, and a basic inventory framework.

A few details on the engine:

- Its  XML-based. with a boolean 'named key' logic scheme, where each 'keyid' is either 1 or 0.
- each page contains core objects (normal rendered & or positional objects)
- as well as fork objects (logic operations, timing, sound playing, 'page' changes, control logic, etc.)
- The engine plugins are based around python classes.
- its primary 'keyid fork' logic is checked in parallel. though a basic timing 'fork' and the animation system can help sequence events.
- the animation system isn't finished. _(i.e. the goto commands are getting label functionality in the next version.)_
- Some example readmes can be found in _manual/example_READMEs_
- value the actual XML code and comments over the manual. Feel free to bring up any questions/issues though. 
- the test*.xml pages have many small tests.
- ENGSYSTEM.xml is the main configuration file. its core and fork sections are active on all pages.

**Comment by fitoprincipe** [comment]
Great! I'll try it =) Thank you

**Comment by fitoprincipe** [comment]
Is your "latest stable release" the master branch? How do you manage the development process? May be I should open another ticket.. :thinking:

**Comment by ThomasTheSpaceFox** [comment]
are you not aware of github's releases system?

https://github.com/ThomasTheSpaceFox/Desutezeoid/releases

**Comment by fitoprincipe** [comment]
Oh yes, I'm talking about this: https://nvie.com/posts/a-successful-git-branching-model/

**Comment by ThomasTheSpaceFox** [comment]
Ive been using the same development process for half a decade.  Some of my projects are so complicated they don't even have a stable release to put in the master branch. i.e. SBTCVM Gen2-9 (which is still in alpha after the greater part of a year)

that aside, i have always kept releases using github's release system, which assigns them tags.

This honestly isn't my highest priority project. SBTCVM is. and my gopherspace client has priority beyond that.  I respect your opinion, don't get me wrong. But i tend to be the guy who sticks with what has worked.

**Comment by fitoprincipe** [comment]
I't fine! I'm just asking =)
